vmm/sandbox/src/bin/cloud_hypervisor/main.rs

```diff
@@ -35,7 +35,11 @@ async fn main() {
     let config = Config::load_config(&args.config).await.unwrap();
 
     // Update args log level if it not presents args but in config.
-    init_logger(&args.log_level.unwrap_or(config.sandbox.log_level()));
+    let enable_tracing = config.sandbox.enable_tracing;
+    let log_level = args.log_level.unwrap_or(config.sandbox.log_level());
+    let service_name = "kuasar-vmm-sandboxer-clh-service";
+    tracer::setup_tracing(&log_level, enable_tracing, service_name).unwrap();
+    tracer::set_enabled(enable_tracing);
```

Comment:
The `ENABLED: AtomicBool` static variable is the only one to detiminate, so it's better to:
```rust
...
tracer::set_enabled(config.sandbox.enable_tracing);
tracer::setup_tracing(&log_level, service_name).unwrap();
```
and let `setup_tracing()` read `ENABLED`

Reply:
Fixed.